test/test_other.py

```diff
@@ -2620,6 +2620,8 @@ def test_embind(self, extra_args):
          '--pre-js', test_file('embind/test.pre.js'),
          '--post-js', test_file('embind/test.post.js'),
          '-sWASM_ASYNC_COMPILATION=0',
+         # The default stack size is not enough for these tests.
```

Comment:
Do you know why that is? Does embind use lots of stack space for some reason? If so, in theory we could consider a different default stack size when `--bind` is set, though I don't know if that's a good idea.

Reply:
This is one of our more heavy weight tests I think.. since it embeds an entire JS test suite inside a single unit test.  I will try to figure out what is going on..